WebApplication/Backend/Repositorys/FeedbackRepository.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Backend.Repository;
+using Model.Blog;
+using System;
+using System.Collections.Generic;
+
+namespace WebApplication.Backend.Repositorys
```

Comment:
You should add this as the interface in the shared backend, just like other repositories - it should inherit the GenericRepository<Feedback>. (You don't need to make a FeedbackFileSystem class though)